src/products/ProductGraphQlRequestBuilder.js

```diff
@@ -25,15 +25,21 @@ class ProductGraphQlRequestBuilder {
         this.requestArgs = requestArgs;
         this.template = template;
         this.context = {};
+        this.getBySku = false;
+    }
+
+    bySku() {
```

Comment:
Can you make this a bit more explicit and add the sku as an attribute? It's a bit strange that you can write `builder.bySku().build()` and the sku is magically parsed from the `args` variable.

Reply:
Will remove that code. See discussion with Sorin.

Reply:
Also, having `builder.build()` is equally strange because *everything* is magically parsed from the `args` variable. Maybe it's time to have a discussion about this approach of just sending `args` to functions and extract at action level what is needed from `args` and have functions defined real arguments.